docs/reference_guides/model_libraries/generic/costing/process_costing.rst

```diff
@@ -0,0 +1,891 @@
+Process Costing Library
+=======================
+
+.. contents:: Contents 
+    :depth: 4
+
+Introduction
+------------
+
+.. rubric:: IDAES Process Costing Framework
+
+This document outlines an example process costing library to support IDAES unit models. For a general overview of costing capabilities in IDAES, see the :ref:`IDAES Process Costing Framework<reference_guides/core/costing/costing_framework:IDAES Process Costing Framework>`.
```

Comment:
I would make this a note block as well - otherwise it might be interpreted as a general introduction